### DISCUSSION TITLE
Is it possible to update Debian to a newer version ?

### DISCUSSION THREAD

**Discussion by user858753257** [utmapp/UTM#2488] [UTM Help (macOS)]
I mean from the prebuild 10.4 to the latest 10.9.x ?

**Comment by ty-yqs** [comment]
Can the VM connect to the internet? If it can, you can follow the following steps.

You must make a backup of the system before you do this. Any upgrade poses application and data loss risk. Therefore keep verified backups.

The procedure is as follows. First, note down the current version:
```
$ lsb_release -a
$ cat /etc/debian_version
$ uname -mrs
```
Type the following apt-get command/apt command to upgrade your system:
```
$ sudo apt-get update
```
OR
```
$ sudo apt update
```
Please note the version number changed in last line. Next, apply updates, run:
```
$ sudo apt-get dist-upgrade
```
OR
```
$ sudo apt dist-upgrade
```
If `/var/run/reboot-required` found it, reboot the box:
```
[ -f /var/run/reboot-required ] && echo "System reboot needed"
```
See “[how to find out if my Ubuntu/Debian Linux server needs a reboot](https://www.cyberciti.biz/faq/how-to-find-out-if-my-ubuntudebian-linux-server-needs-a-reboot/)” for more info. Hence, reboot the Linux system, type:
```
$ sudo shutdown -r now
```
Verify that upgrade went smoothly with help of cat command/grep command/egrep command:
```
$ uname -mrs
$ lsb_release -a
$ dmesg | egrep -i 'err|warn|critical'
$ sudo tail -f /var/log/nginx/access_log
```
The installer has been updated to include the fixes incorporated into stable by the point release. You can download updated ISO for full installation too. This stable update added important package corrections and security enhancements to the system.

 Happy upgrades!

**Comment by ty-yqs** [comment]
If the VM can not reach the internet, you could try this.

```
# DNS
vim /etc/dhcp/dhclient.conf
# append
supersede domain-name-servers 8.8.8.8;
supersede domain-name "";
supersede domain-search "";
root@Debian:~# systemctl restart networking.service
root@Debian:~# cat /etc/resolv.conf
nameserver 8.8.8.8
```

**Reply by user858753257** [reply]
Thanks . But how can I connect to the internet ? 

When I take the default values I didn't get network

**Reply by ty-yqs** [reply]
Choose the network card named `virtio` in the VM settings.

**Reply by user858753257** [reply]
@ty-yqs 

Which "virtio" option do you mean ? 
<img width="625" alt="Bildschirmfoto 2021-05-08 um 19 34 41" src="https://user-images.githubusercontent.com/8161283/117548310-96386b80-b034-11eb-9eaa-5cf4aa519e0c.png">

**Reply by ty-yqs** [reply]
Every `virtio` option is OK.

**Reply by user858753257** [reply]
Hmm . None of them are working 😏

**Reply by ty-yqs** [reply]
> Hmm . None of them are working 😏

You might ask others for help. It’s working on my iPad

**Reply by user858753257** [reply]
maybe there is a difference between iPad and Mac

**Reply by rvgeerligs** [reply]
I have the same problem with MacBook M1. Did you ever solve this?
